src/lib/FormGroupLayout.tsx

```diff
@@ -1,37 +1,63 @@
-import { PropsWithChildren } from "react";
+import { PropsWithChildren, useState } from "react";
 import { FieldValues, useFormContext } from "react-hook-form";
-import { FormGroup, Label, FormFeedback, FormText } from "reactstrap";
+import { FormGroup, Label, FormFeedback, FormText, Tooltip } from "reactstrap";
 import { useSafeNameId } from "src/lib/hooks/useSafeNameId";
 import { CommonInputProps } from "./types/CommonInputProps";
+import "./styles/FormGroupLayout.css"
 
 interface FormGroupLayoutProps<T extends FieldValues>
-  extends PropsWithChildren<Pick<CommonInputProps<T>, "helpText" | "label" | "name" | "id">> {
-  layout?: "checkbox" | "switch";
+    extends PropsWithChildren<Pick<CommonInputProps<T>, "helpText" | "label" | "name" | "id" | "labelToolTip">> {
+    layout?: "checkbox" | "switch";
 }
 
 const FormGroupLayout = <T extends FieldValues>(props: FormGroupLayoutProps<T>) => {
-  const { label, helpText, children, layout } = props;
-  const { name, id } = useSafeNameId(props.name, props.id);
-  const {
-    formState: { errors },
-  } = useFormContext();
+    const { label, helpText, children, layout, labelToolTip } = props;
```

Comment:
it looks like this is formatted differently? 

But there is a check in the `CI` that will check if its formatted according to prettier. Just as a headsup.

Reply:
Hm, I do use the prettier formatter. I formatted it again and it should know be as intended.